examples/main/src/layer-examples.js

```diff
@@ -241,14 +242,36 @@ const HexagonLayerExample = {
     data: dataSamples.hexagons,
     hexagonVertices: dataSamples.hexagons[0].vertices,
     getColor: h => [48, 128, h.value * 255, 255],
-    getElevation: h => h.value * 50,
+    getElevation: h => h.value * 5000,
     extruded: true,
     pickable: true,
-    radiusScale: 1,
     opacity: 1
   }
 };
 
+const PointDensityHexagonLayerExample = {
+  layer: PointDensityHexagonLayer,
+  props: {
+    id: 'PointDensityHexagonLayer',
+    data: dataSamples.points,
+    extruded: true,
+    pickable: true,
+    radius: 1000,
+    opacity: 1,
+    elevationScale: 0.8,
+    coverage: 1,
+    getPosition: d => d.COORDINATES,
+    lightSettings: {
```

Comment:
Should we use a global object for `lightSettings` so that all layers in the example get lit the same way?